Snakefile

```diff
@@ -184,6 +228,25 @@ rule parameter_advise:
     input: expand('{out_dir}{sep}pathway-{dataset}-{algorithm}-{params}.txt', out_dir=out_dir, sep=os.sep, dataset=datasets, algorithm=algorithms, params=pathlinker_params)
     output: os.path.join(out_dir, 'advised-pathway-{dataset}-{algorithm}.txt')
     shell: 'echo {input} >> {output}'
+'''
+
+# Collect Summary Statistics
+rule summarize:
+    input:
+        standardized_file = os.path.join(out_dir, 'pathway-{dataset}-{algorithm}-{params}.txt')
+    output:
+        summary_file = os.path.join(out_dir, 'pathway-{dataset}-{algorithm}-{params}-summary.txt')
+    run:
+        summary.run(input.standardized_file,output.summary_file,directed=algorithm_directed[wildcards.algorithm])
+
+# Write GraphSpace JSON Graphs
+rule viz_graphspace:
+    input: standardized_file = os.path.join(out_dir, 'pathway-{dataset}-{algorithm}-{params}.txt')
+    output:
+        graph_json = os.path.join(out_dir, 'pathway-{dataset}-{algorithm}-{params}-gs.json'), style_json = os.path.join(out_dir, 'pathway-{dataset}-{algorithm}-{params}-gs-style.json')
```

Comment:
We might have future ones though...to be consistent I think prefixes are the way to go.

Reply:
In the future we may want to move to subdirectories instead of stacking prefixes, but that would be an easy change to make at a later time if we want it.

Reply:
Sounds good. The JSON filename pattern has been updated in commit 4e75eeb